src/groups/circle_group.jl

```diff
@@ -12,7 +12,7 @@ show(io::IO, ::CircleGroup) = print(io, "CircleGroup()")
 
 has_invariant_metric(::CircleGroup, ::ActionDirection) = Val(true)
 
-is_default_metric(::MetricManifold{CircleGroup,EuclideanMetric}) = Val(true)
+is_default_metric(::MetricManifold{CircleGroup,EuclideanMetric}) = true
```

Comment:
I think we should keep the THTT pattern for decorators and return `Val(true)` here.